public/scripts/models/movieModel.js

```diff
@@ -1 +1,20 @@
 'use strict';
+
+var app = app || {};
+
+(function(module){
+  var movieModel = {
+    movieInputs: [],
```

Comment:
Is movieInputs coming to the ajax call to server? or is it title input from user? perhaps a comment?